Sources/CwlViews/Implementations/iOS/CwlPickerView_iOS.swift

```diff
@@ -0,0 +1,370 @@
+		public var inherited = Inherited()
+		public var dynamicDelegate: Delegate? = nil
+		public let delegateClass: Delegate.Type
+		public init(delegateClass: Delegate.Type) {
+			self.delegateClass = delegateClass
+		}
+
+		public func constructStorage(instance: Instance) -> Storage { return Storage() }
+		public func inheritedBinding(from: Binding) -> Inherited.Binding? {
+			if case .inheritedBinding(let b) = from { return b } else { return nil }
+		}
+	}
+}
+
+// MARK: - Binder Part 4: Preparer overrides
+public extension PickerView.Preparer {
+	var delegateIsRequired: Bool { return true }
+
+	mutating func prepareBinding(_ binding: Binding) {
+		switch binding {
+		case .inheritedBinding(let x): inherited.prepareBinding(x)
+		case .didSelectRowAndComponent(let x): delegate().addMultiHandler2(x, #selector(UIPickerViewDelegate.pickerView(_:didSelectRow:inComponent:)))
+		default: break
+		}
+	}
+
+	func prepareInstance(_ instance: Instance, storage: Storage) {
+		inheritedPrepareInstance(instance, storage: storage)
+
+		prepareDelegate(instance: instance, storage: storage)
+		instance.dataSource = storage
+	}
+
+	func applyBinding(_ binding: Binding, instance: Instance, storage: Storage) -> Lifetime? {
+		switch binding {
+		case .inheritedBinding(let x): return inherited.applyBinding(x, instance: instance, storage: storage)
+
+		// 1.
+		case .showsSelectionIndicator(let x): return x.apply(instance) { i, v in i.showsSelectionIndicator = v }
+		case .pickerData(let x): return x.apply(instance, storage) { i, s, v in s.applyComponents(v, to: i) }
+
+		// 2.
+		case .selectRowAndComponent(let x): return x.apply(instance) { i, v in i.selectRow(v.value.row, inComponent: v.value.component, animated: v.isAnimated) }
+		case .reload(let x): return x.apply(instance) { i, v in
+			switch v {
+			case .all:
+				i.reloadAllComponents()
+			case .component(let component):
+				i.reloadComponent(component)
+			}
+		}
+
+		// 4. Delegate Bindings
+		case .didSelectRowAndComponent(_): return nil
+		case .viewConstructor(let x):
+			storage.viewConstructor = x
+			return nil
+		}
+	}
+}
+
+// MARK: - Binder Part 5: Storage and Delegate
+extension PickerView.Preparer {
+
+	open class Storage: View.Preparer.Storage, UIPickerViewDelegate, UIPickerViewDataSource {
+		open override var isInUse: Bool { return true }
+
+		open var components = PickerComponentState<PickerComponent<ViewData>>()
+		open var viewConstructor: ((SignalMulti<ViewData>) -> ViewConvertible)?
+
+		// Data Source
+		open func numberOfComponents(in pickerView: UIPickerView) -> Int {
+			return components.count
+		}
+
+		open func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+			return components.at(component)?.elements.count ?? 0
+		}
+
+		// Delegate
+		open func pickerView(_ pickerView: UIPickerView, viewForRow row: Int, forComponent component: Int, reusing view: UIView?) -> UIView {
+			if let element = components.at(component)?.elements.at(row) {
```

Comment:
It would be good to somehow introduce view reuse for efficiency. But I got a little stuck with the implementation.